src/Table/Column/Checkbox.php

```diff
@@ -37,6 +37,7 @@ protected function init(): void
         if (!$this->class) {
             $this->class = 'cb_' . $this->shortName;
         }
+        $this->getApp()->requireJs($this->getApp()->cdn['atk'] . '/../js/src/helpers/grid-checkbox.helper.js');
```

Comment:
This line should be removed.
The javascript should be compiled (how?) in such a way it is included in the standard atk4 javascripts.
 Should be called after `$('#atk_layout_maestro_grid_view_table').find('.cb_table_column_checkbox').checkbox();`

![image](https://user-images.githubusercontent.com/8080934/200188425-143ee631-300b-4379-bb2b-68eb1a1f7edb.png)

 How to make this atk4/ui friendly ?

Reply:
Ok I should invoke the power of
```Bash
npm install
npm run dev
npm run build
```